ui/apps/akasha/src/extensions/image-block/image-editor-block.tsx

```diff
@@ -167,10 +162,28 @@ export const ImageEditorBlock = (
     [createBlock, retryCreate],
   );
 
+  const isImgUrl = async url => {
+    try {
+      const response = await fetch(url, { method: 'HEAD' });
+      const isImg = response?.headers?.get('Content-Type')?.startsWith('image');
+      return isImg;
+    } catch (error) {
+      setURLNotImage(true);
+      const notifMsg = error?.message;
+      _uiEvents.current.next({
+        event: NotificationEvents.ShowNotification,
+        data: {
+          type: NotificationTypes.Error,
+          message: notifMsg,
+        },
+      });
+    }
+  };
+
   const handleChange = async e => {
```

Comment:
```typescript
const handleChange = async (e) => {
  const imageUrl = e.currentTarget.value;
  setImageLink(imageUrl);
  try {
    const isImg = await isImgUrl(imageUrl);
    setURLNotImage(!isImg);
    if (!isImg) {
      const notifMsg = t(`URL doesn't contain an image.`);
      _uiEvents.current.next({
        event: NotificationEvents.ShowNotification,
        data: {
          type: NotificationTypes.Error,
          message: notifMsg,
        },
      });
    }
  } catch (error) {
    const notifMsg = error.message;
    _uiEvents.current.next({
      event: NotificationEvents.ShowNotification,
      data: {
        type: NotificationTypes.Error,
        message: notifMsg,
      },
    });
  }
};
```

Reply:
I see now, you are catching the error here, I will test it